visualization/app/codeCharta/ui/dialog/dialog.addScenarioSettings.component.html

```diff
@@ -0,0 +1,28 @@
+<md-dialog class="scenario-dialog">
+	<md-toolbar>
+		<div class="md-toolbar-tools">
+			<h2>Add Scenario</h2>
+		</div>
+	</md-toolbar>
+
+	<md-dialog-content>
+		<div class="md-dialog-content">
+			<md-input-container class="md-block">
+				<label>Scenario Name</label>
+				<input class="filename-input" type="text" ng-model="$ctrl._viewModel.scenarioName" />
```

Comment:
This is not a matching CSS class name. Either rename to `scenario-name` and add in scss file or rename to `dialog-input` if this matches with the other usages